lib/fetch/index.js

```diff
@@ -105,7 +102,7 @@ async function fetch (...args) {
     args[1] !== undefined
   ) {
     throw new TypeError(
-      "Failed to execute 'fetch' on 'Window': cannot convert to dictionary."
+      "Failed to execute 'fetch': cannot convert to dictionary."
```

Comment:
This is externally observable. I would prefer if the error is the exact same as in Chrome.